.github/workflows/tests_secondary.yml

```
@@ -288,3 +288,24 @@ jobs:
         flakiness-subscription-id: ${{ secrets.AZURE_FLAKINESS_DASHBOARD_SUBSCRIPTION_ID }}
       env:
         PWTEST_CHANNEL: chromium-headless-shell
+
+  test_chromium_next:
+    name: Test chromium-next channel
+    environment: ${{ github.event_name == 'push' && 'allow-uploading-flakiness-results' || null }}
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on: [ubuntu-latest, windows-latests, macos-latest]
+    runs-on: ${{ matrix.runs-on }}
+    steps:
+    - uses: actions/checkout@v4
+    - uses: ./.github/actions/run-test
+      with:
+        browsers-to-install: chromium
```

shouldn't this install chromium-next?

```suggestion
        browsers-to-install: chromium-next
```

---

`chromium` is needed for codegen tests that run it headlessly. I have manually verified that everything works for the end user.